webapp/templates/purl-create.xhtml

```diff
@@ -116,7 +103,29 @@
             <div class="col-sm-4">
                 <aside class="well">
                     <p>PURLs are used to curate the URL resolution process, thus solving
-                    the problem of transitory URIs in location-based URI schemes like HTTP</p>
+                    the problem of transitory URIs in location-based URI schemes like HTTP.</p>
+                    <p>This class of PURL deals with cases where there are alternative data representations describing a resource or cases where a resource is unavailable.</p>
+                    <dl>
+                        <dt>Copy (200)</dt>
+                        <dd>Expose content from one location to another, possibly including limiting access to read-only</dd>
+                        <dt>Canonical (301)</dt>
+                        <dd>Moved permanently to a target URL</dd>
+                        <dt>Alternate (302)</dt>
+                        <dd>Simple redirection to a target URL</dd>
+                        <dt>Described by (303)</dt>
+                        <dd>See other URL, indicates that the target is data describing some other resource, maybe in the real world</dd>
+                        <dt>Resides (307)</dt>
+                        <dd>Indicates that the redirection is temporary</dd>
+                        <dt>Moved (308)</dt>
+                        <dd>Indicates that the target URL has been permanently moved</dd>
+                        <dt>Missing (404)</dt>
+                        <dd>Indicates that the target URL is unavailable, which might be temporary</dd>
+                        <dt>Gone (410)</dt>
+                        <dd>Indicates that the target URL is permanently gone</dd>
+                        <dt>Illegal (451)</dt>
+                        <dd>Intended to be used when resource access is denied for legal reasons, e.g. censorship or government-mandated blocked access</dd>
+                    </dl>
+                    <p>For more advanced features use a RewriteRule instead.</p>
```

Comment:
Don't link or talk about more advanced classes here, user might not have that as an option. However, the docs is a good place for that.